Security/Authorization/Interception/MethodSecurityInterceptor.php

```diff
@@ -50,11 +52,15 @@ class MethodSecurityInterceptor implements MethodInterceptorInterface
     private $runAsManager;
     private $logger;
 
-    public function __construct(SecurityContextInterface $securityContext, AuthenticationManagerInterface $authenticationManager, AccessDecisionManagerInterface $accessDecisionManager,
+    public function __construct($tokenStorage, AuthenticationManagerInterface $authenticationManager, AccessDecisionManagerInterface $accessDecisionManager,
```

Comment:
Maybe we can add a doc comment what to expect here, i.e. security context or token storage?